climada/hazard/test/test_base.py

```diff
@@ -1444,32 +1450,47 @@ def test_get_paa(self):
     def test_get_fraction(self):
         haz = dummy_hazard()
 
+        #standard index
         idx = [0, 1]
         cent_idx = np.array(idx)
-        frac = haz.get_fraction(cent_idx)
+        frac = haz._get_fraction(cent_idx)
         true_frac = haz.fraction[:, idx]
         np.testing.assert_array_almost_equal(frac.toarray(), true_frac.toarray())
 
         #repeated index
         idx = [0, 0]
         cent_idx = np.array(idx)
-        frac = haz.get_fraction(cent_idx)
+        frac = haz._get_fraction(cent_idx)
         true_frac = haz.fraction[:, idx]
         np.testing.assert_array_almost_equal(frac.toarray(), true_frac.toarray())
 
+        #index is None
+        cent_idx = None
+        frac = haz._get_fraction(cent_idx)
+        true_frac = haz.fraction
+        np.testing.assert_array_almost_equal(frac.toarray(), true_frac.toarray())
```

Comment:
Is there a reason why you cannot call `assert_array_equal`? There are no calculations involved, only copying and slicing, and therefore the values should be *exactly* the same, right? (Same for the other two assertions above)
```suggestion
        np.testing.assert_array_equal(frac.toarray(), true_frac.toarray())
```